api/app_sph_lms/models.py

```diff
@@ -1,379 +1,448 @@
+        db_table = "app_sph_lms_classes"
+
+    def __str__(self):
+        return str(self.name)
+
+
+class Trainer(models.Model):
+    class_id = models.ForeignKey(Class, on_delete=models.CASCADE, null=True)
+    trainer = models.ForeignKey(User, on_delete=models.CASCADE)
+    company = models.ForeignKey(
+            Company,
+            on_delete=models.CASCADE,
+            related_name="trainer"
+        )
+    status = models.ForeignKey(Status, on_delete=models.CASCADE, default=1)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        unique_together = ('class_id', 'trainer')
+        verbose_name = "Trainer"
+        verbose_name_plural = "Trainer"
+        db_table = "app_sph_lms_trainers"
+
+    def __str__(self):
+        return (
+            "Company: " +
+            str(self.company) +
+            " | " +
+            "Trainer: " +
+            str(self.trainer) +
+            " | " +
+            "Class: " +
+            str(self.class_id)
+        )
+
+
+class Trainee(models.Model):
+    class_id = models.ForeignKey(
+            Class,
+            on_delete=models.CASCADE,
+            null=True
+        )
+    trainee = models.ForeignKey(
+            User,
+            on_delete=models.CASCADE,
+            related_name="user"
+        )
+    company = models.ForeignKey(
+            Company,
+            on_delete=models.CASCADE,
+            related_name="trainee"
+        )
+    status = models.ForeignKey(Status, on_delete=models.CASCADE, default=1)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        unique_together = ('class_id', 'trainee')
+        verbose_name = "Trainee"
+        verbose_name_plural = "Trainee"
+        db_table = "app_sph_lms_trainees"
+
+    def __str__(self):
+        return (
+            "Company: " +
+            str(self.company) +
+            " | " +
+            "Trainee: " +
+            str(self.trainee) +
+            " | " +
+            "Class: " +
+            str(self.class_id)
+        )
+
+
+class Course(models.Model):
+    company = models.ForeignKey(
+            Company,
+            on_delete=models.CASCADE,
+            related_name="company",
+            null=True,
+            blank=True
+        )
+    author = models.ForeignKey(
+            User,
+            on_delete=models.CASCADE,
+            related_name="author",
+            null=True,
+            blank=True
+        )
+    category = models.ManyToManyField(Category, related_name='categories')
+    name = models.CharField(max_length=255, validators=[MinLengthValidator(3)])
+    description = models.TextField(
+            max_length=65000,
+            null=True,
+            validators=[MinLengthValidator(5)]
+        )
+    is_active = models.BooleanField(default=True)
+    image = models.CharField(max_length=255, null=True)

```

Comment:
If you are gonna change the course based on ERD, I'm afraid you are gonna have to change this in FE as well, some pages must be expecting img_path on FE, and currently broken, one of them is 'trainer/courses/' page, please check the others as well